tests/unit/test_sql_sensor.py

```diff
@@ -110,8 +110,7 @@ def tearDownClass(cls) -> None:
 
 class ClickHouseLegacySqlSensorTestCase(unittest.TestCase):
     @mock.patch(
-        'airflow_clickhouse_plugin.ClickHouseSqlSensor'
-            '._ClickHouseSqlSensor__poke_clickhouse',
+        'airflow_clickhouse_plugin.sensors.clickhouse_sql_sensor.ClickHouseSqlSensor'
```

Comment:
trailing comma

Reply:
Also please fit this code line into 80 characters width, e.g.

```suggestion
        'airflow_clickhouse_plugin.sensors.clickhouse_sql_sensor'
            '.ClickHouseSqlSensor',
```

Reply:
No need anymore since the patch is removed now.